src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const personPartnerParagraph = person.isMarried ? (
+    <p className="Person__partner">
+      {person.partnerName}
+      {' '}
+      is my
+      {' '}
+      {person.sex === 'm' ? 'wife' : 'husband'}
```

Comment:
Use template strings.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
Check all places in project

Reply:
Done